documentation/Database-Schema-Review-And-Consolidations.md

```diff
@@ -0,0 +1,122 @@
+
+## Review Levels
+
+Each stage of application is broken down into review levels, base level (1) is the review of application, level > 1 are reviews of reviews (consolidations). Level 1 reviewer reviews application (including re-reviews as a result of applicant replying to LOQ changes), and deals with changes requested from consolidator. Likewise consolidators review reviews and make suggested decision
+
+## Review Decision
+
+Only available on level > 1 reviewer or when `is_last_level` in `review_assignment` is set. Consolidators can make/suggest decisions. If there is only one review level (say for screening), then the screener has the ability to create review decisions
+
+`Conform (Approve)` -> All of the applicant's responses must be marked as approved by reviewer, and consolidator agreed with reviewers decision
+
+`LOQ` -> At least one of applicant's responses marked as non conformity (Decline) and consolidator agreed with reviewers decisions. They have an option to select from LOQ or Non-Conformity
+
+`Non-conofrm (Decline` -> At least one of applicant's responses marked as non conformity (Decline) and consolidator agreed with reviewers decisions. They have an option to select from LOQ or Non-Conformity
+
+`Changes Requested` -> Applies to consolidation, if at least one disagreement with reviewer below consolidator's level, this is the only option available
+
+## Review Assignment
+
+These records are created by back end when review level is reached (either through application submission or review submission, see Review Life Cycle diagram below). `Template Permission` records, alongside `permission name`, `permission name join` and `permission policy` are used to created relevant `review assignments`
+They can be queried to:
+
+- Determine who can be assigned to an application stage and level <- front end task
+- Determine if an application stage and level is fully assigned (using review_question_assignment link) <- through application list view
+- Determine if review can be started/created <- front end can check if review can be created (including self assignment) See `Review and Consolidation URL flow` diagram below
+
+#### Review Assignment Status
+
+`Available` -> can be assigned by assigner
+`Not Available` -> cannot be assigned by assigner, in case of `available for self assignment`, when reviewer self assigns a review, all other `review assignments` for current `stage` and `level` are changed to this status
+`Assigned` -> review is assigned to a user and can be started (in case of self assignment when review is self assigned, review assignment will changes status to assigned)
+`Available for self assignment` -> review can be started and self assigned
+
+`Assignment NOTE` -> For MVP, only level 1 reviewer can be assigned questions (vs self assignment of review), level > 1 will be self assigned (level 1 can also be configured to self assigned, in this case they will need to be configured to review all sections)
```

Comment:
These need either double-spaces at the end of each to force a line-break, or space between to make separate paragraphs. Currently they render all in one continuous paragraph.

Reply:
Oops, I should have been more diligent in checking formatting:

85e16cb2a3d5923c19c556128122fd2fcc9a2ece